xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -439,14 +439,24 @@
       #set($discard = $itemDoc.use($className))
       #set($discard = $row.put('doc_objectCount', $itemDoc.getObjectNumbers($className)))
       #set($discard = $row.put('doc_edit_url', $itemDoc.getURL($itemDoc.defaultEditMode)))
-      #set($discard = $row.put('doc_author_url', $xwiki.getURL($translatedDoc.author)))
       #set($discard = $row.put('doc_date', $xwiki.formatDate($translatedDoc.date)))
       #set($discard = $row.put('doc_title', $translatedDoc.plainTitle))
       #set($rawTitle = $translatedDoc.title)
       #if($rawTitle != $row['doc_title'])
         #set($discard = $row.put('doc_title_raw', $rawTitle))
       #end
-      #set($discard = $row.put('doc_author', $xwiki.getPlainUserName($translatedDoc.authorReference)))
+      ## Note that this serialization step is mandatory and we cannot count on the UserReferenceConverter here
+      ## because UserReference#toString does not rely on the serializer so it's not the expected format for
```

Comment:
I don't understand the relationship with UserReference#toString, why don't you simply pass the reference to #getPlainUserName (and it will be converted) ?

Reply:
Hmmm I put the comment we already had in contentheader.vm but there I was using `getUserName` which takes a String as parameter and here I'm using `getPlainUserName` which takes a DocumentReference so indeed I might not need this.

Reply:
Even if it was taking a String, you would have a real convertion to String and not a toString() call